css/css-writing-modes-3/table-cell-001.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<html lang="en">
+<meta charset="utf-8">
+<title>CSS Writing Mode test: orthogonal table cell</title>
+<link rel="help" href="https://drafts.csswg.org/css-writing-modes-3/#block-flow">
+<meta assert="The writing mode property applies to table cells: two adjacent cells can have orthogonal writing modes, and the enclosing table must be laid out correctly.">
+<meta flags="ahem">
+<link rel="match" href="reference/table-cell-001-ref.html">
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+
+<style>
+@font-face {
```

Comment:
Yes, please remove it.

Where are the other versions of Ahem around? We should only have `css/fonts/ahem/ahem.ttf` nowadays (`css/fonts/ahem-extra` is… different).